src/servers/src/postgres/types.rs

```diff
@@ -351,6 +352,23 @@ fn encode_array(
     }
 }
 
+pub fn convert_err(e: impl ErrorExt) -> PgWireError {
```

Comment:
It's just a little weird to have this in types.rs. We can create a new module like utils or common in postgres for this.

Reply:
copy